### PR TITLE
Fix lint across pilot and mixer

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: pilot/pkg/bootstrap/server.go [hunk 1]
```diff
@@ -22,10 +22,10 @@ import (
 
 	"code.cloudfoundry.org/copilot"
 	"github.com/davecgh/go-spew/spew"
+	multierror "github.com/hashicorp/go-multierror"
```
Comment (nmittler) [inline pilot/pkg/bootstrap/server.go]: This alias should not be necessary.  Is there a reason it was added?

**Comment by wattli** [comment]
This change is auto generated by gofmt and goimports

File: pilot/pkg/bootstrap/server.go (hunk 1 above)
Comment (wattli) [reply pilot/pkg/bootstrap/server.go]: it's auto generated by gofmt.

**Comment by nmittler** [comment]
@wattli ok, fair enough ... thanks

**Comment by wattli** [comment]
@nmittler @ayj , the local running make lint returns okay (see logs below), but the circleci lint still complains, any ideas?

...
Running gometalinter on adapters ....
+ pushd mixer/tools/adapterlinter
~/Desktop/src/istio.io/istio/mixer/tools/adapterlinter ~/Desktop/src/istio.io/istio
+ go install .
+ popd
~/Desktop/src/istio.io/istio
+ gometalinter.v2 --config=./mixer/tools/adapterlinter/gometalinter.json ./mixer/adapter/...
+ echo 'gometalinter on adapters OK'
gometalinter on adapters OK

**Comment by ijsnellf** [comment]
@wattli: I think I hit that recently. Make sure you don't have an old version of gometalinter installed in $GOPATH/bin or in $GOPATH/src.

**Pull request merged**